src/pages/ListScholarships.jsx

```diff
@@ -56,38 +58,41 @@ function ListScholarships() {
       </Helmet>
       <Drawer
         sx={{
-          display: { xs: 'none', md: 'block' },
           flexShrink: 0,
           width: drawerWidth,
           '& .MuiDrawer-paper': {
             width: drawerWidth,
             position: 'absolute',
             boxSizing: 'border-box',
           },
-          position: 'sticky',
+          position: { xs: 'static', md: 'sticky' },
           overflowY: 'auto',
         }}
-        variant="permanent"
+        open={drawerOpen || isDesktop}
+        variant={isDesktop ? 'permanent' : 'temporary'}
         anchor="left">
-        <FilterPanel />
+        <FilterPanel onClose={() => setDrawerOpen(false)} />
       </Drawer>
-      <Container
-        maxWidth="md"
-        component="main"
-        ref={ref}
+
+      <Box
         sx={{
-          bgcolor: 'background.default',
-          flexGrow: 1,
-          padding: 2,
-          position: 'sticky',
-          overflowY: { md: 'auto' },
+          width: '100%',
+          display: 'flex',
+          flexDirection: 'column',
         }}>
-        <Typography variant="h4" component="h1" align="center" style={{ p: 1 }}>
-          {t('general.scholarships')}
-        </Typography>
-        <FilterBar />
-        <ScholarshipList filters={queryFilters} />
-      </Container>
+        <FilterBar openFilter={() => setDrawerOpen(true)} />
+
+        <Container
+          maxWidth="md"
+          component="main"
+          ref={ref}
+          sx={{
+            flexGrow: 1,
+            overflowY: 'auto',
```

Comment:
Didn't add to Box above so `FilterBar` could scroll with the list of scholarships.
Also made it auto so it could scroll when on mobile.

Reply:
I don't see the FilterBar scrolling with the scholarships?
<img width="553" alt="Screen Shot 2022-02-02 at 9 08 20 PM" src="https://user-images.githubusercontent.com/8023233/152284732-6b1efa92-38d3-4b5a-845d-bdccf7caba57.png">

I think instead of here you want to set `overflowY` in the parent `Box`